Options.py

```diff
@@ -23,6 +23,12 @@
     import pathlib
 
 
+def roll_percentage(percentage: typing.Union[int, float]) -> bool:
```

Comment:
```suggestion
def roll_percentage(percentage: int | float) -> bool:
```
Python 3.8 and 3.9 gone